src/tools/rbd_mirror/image_sync/SnapshotCopyRequest.h

```diff
@@ -104,6 +104,7 @@ class SnapshotCopyRequest : public BaseRequest {
   librados::snap_t m_prev_snap_id = CEPH_NOSNAP;
 
   std::string m_snap_name;
+  cls::rbd::SnapshotNamespace m_snap_namespace;
```

Comment:
It was more of a comment that the ```SnapshotCopyRequest``` state machine should skip over any snapshots that aren't user snapshots. I think the ```m_remote_snap_ids``` and ```m_local_snap_ids``` collections should be filtered to only include user snapshots.